tests/unit/TooltipTest.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {screen, render, fireEvent} from '@testing-library/react';
+import ContextMenuItem from '../../src/components/ContextMenuItem';
+
+const Localize = require('../../src/libs/Localize');
+
+test('Show text in tooltip', () => {
+    render(<ContextMenuItem
+        icon=""
+        text={Localize.translate('en', 'reportActionContextMenu.copyToClipboard')}
+        successIcon=""
+        successText={Localize.translate('en', 'reportActionContextMenu.copyToClipboard')}
+        onPress={() => { }}
+    />);
+    const element = screen.getByText(Localize.translate('en', 'reportActionContextMenu.copyToClipboard'));
+    fireEvent.mouseOver(element);
+
+    screen.findByText(Localize.translate('en', 'reportActionContextMenu.copyToClipboard')).then((result) => {
+        expect(
+            result,
+        ).toBe(element);
+    });
+});
```

Comment:
This existing test is good, but we need to add another separate test the actual change you're making. So this probably means something like this:

1. Create a manual mock for `matchMedia` in a separate file in `__mocks__`
1. Import that manual mock in this test file
1. Add a separate test case called `Tooltip does not show on devices without a mouse`
1. Mock `matchMedia` to return `false` for the `(pointer:fine)` media query
1. Mouse over the element
1. Assert that the tooltip text is not shown
1. Mock `matchMedia` to return `true` for the `(pointer:fine)` media query
1. Mouse over the element again
1. Assert that the tooltip text is shown

Reply:
You've so far only done steps 1-3 here. Please read over these steps carefully because this lays our pretty closely what the test should do.